rhel.Dockerfile

```diff
@@ -11,22 +11,27 @@
 
 # UPSTREAM: use devtools/go-toolset-rhel7 image so we're not required to authenticate with registry.redhat.io
 # https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhel8/go-toolset
```

Comment:
This URL does not match with the rhel7 go-toolset.

Should be https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/devtools/go-toolset-rhel7

Note also there's a newer (with fewer CVE issues) version of go-toolset-rhel7 too. Might as well take the latest while you're updating this file.